### PR TITLE
Update storage.md

### PR DESCRIPTION
**Pull request by pascaljb**
Updating the section in Akord in line with recent updates to our app, API and docs.

### CODE REVIEW

**Comment by lsgunnlsgunn** [comment]
Hi @pascaljb, I put in the requested changes without knowing there was a PR open for it (I was given a link to a Google doc with the changes instead). Sorry for any confusion. Closing this PR as no longer necessary.

**Comment by pascaljb** [comment]
Hi @lsgunnlsgunn these changes are the latest, in line with very recent updates to the app and API. I believe the Google docs is from a few months ago and the content is out of date. Could we update with this content for the Akord section? Thanks

File: packages/apps/docs/src/docs/build/nft-marmalade/storage.md [hunk 1]
```diff
@@ -123,44 +123,82 @@ This function requires a token ID as its argument and returns the associated URI
...
-Arweave is a pioneering protocol designed to modernize the internet's
-foundational structures, augmenting its capacity for storing, addressing, and
-accessing data. 
-This technology is central to our approach to dealing with NFT
-data, ensuring its perpetual availability and integrity and thereby
-strengthening the overall dependability of the NFTs hosted on Marmalade.
+**Akord** provides a web app, API, CLI and SDK to easily upload and manage data on Arweave. To use Arweave for storing your NFT assets, take the following steps.
 
-### What is Akord?
+### 1. Upload the image asset to Arweave
```
Comment (lsgunnlsgunn) [inline packages/apps/docs/src/docs/build/nft-marmalade/storage.md:132]:
```suggestion
### Upload assets with the Akord app
```

I might suggest getting rid of empty headings and bump up the content a heading level and use numbered steps for steps.

File: packages/apps/docs/src/docs/build/nft-marmalade/storage.md [hunk 2]
```diff
@@ -123,44 +123,82 @@ This function requires a token ID as its argument and returns the associated URI
...
 
-Akord is a platform built on the Arweave blockchain that helps NFT projects and creators get the most out of permanent storage and data ownership. 
-Akord facilitates easy uploading to Arweave, caching, reposting, file management, end-to-end encryption, notifications, token-gated access, private messaging, note creation and more. 
+**1a. Upload with the Akord app**
```
Comment (lsgunnlsgunn) [inline packages/apps/docs/src/docs/build/nft-marmalade/storage.md:134]:
```suggestion
```
This doesn't quite seem necessary to me.

File: packages/apps/docs/src/docs/build/nft-marmalade/storage.md [hunk 3]
```diff
@@ -123,44 +123,82 @@ This function requires a token ID as its argument and returns the associated URI
...
 
-Everything in Akord happens through public or private digital vaults, where you can invite others and set access control. 
+You get 100 MB free on sign up and can easily upload in a few minutes using the web app. [Sign up](https://v2.akord.com/signup), login and follow the instructions to create an “NFT assets” vault.
```
Comment (lsgunnlsgunn) [inline packages/apps/docs/src/docs/build/nft-marmalade/storage.md:136]:
```suggestion
You get 100 MB free on sign up and can easily upload in a few minutes using the web app. 

1. [Sign up](https://v2.akord.com/signup), login and follow the instructions to create an “NFT assets” vault. 
```

File: packages/apps/docs/src/docs/build/nft-marmalade/storage.md [hunk 4]
```diff
@@ -123,44 +123,82 @@ This function requires a token ID as its argument and returns the associated URI
...
 
-Everything in Akord happens through public or private digital vaults, where you can invite others and set access control. 
+You get 100 MB free on sign up and can easily upload in a few minutes using the web app. [Sign up](https://v2.akord.com/signup), login and follow the instructions to create an “NFT assets” vault. 
 
-### How is Akord different?
-While there are other products that also provide data storage on Arweave, Akord has some key differentiating features.
+![Kadena Docs-1](https://github.com/kadena-community/kadena.js/assets/57722629/075c3c75-2a79-4ffa-b715-435c93b63a0c)
 
-- **The easiest and cheapest way to upload to Arweave**. Akord makes it simple to upload to Arweave via the [API](https://docs.akord.com/api-and-dev-tools/quickest-way-to-upload-to-arweave), [CLI](https://docs.akord.com/api-and-dev-tools/build/cli) or [app](https://v2.akord.com/signup). The only service to offer free Arweave storage, 100 MB on [signup](https://v2.akord.com/signup), their prices are also the [cheapest in the ecosystem](https://akord.com/pricing).
+Upload your file to the vault and wait for it to be successfully committed on the Arweave blockchain, normally 5-15 minutes.
```
Comment (lsgunnlsgunn) [inline packages/apps/docs/src/docs/build/nft-marmalade/storage.md:140]:
```suggestion
2. Upload your file to the vault and wait for it to be successfully committed on the Arweave blockchain, normally 5-15 minutes. 
```

File: packages/apps/docs/src/docs/build/nft-marmalade/storage.md [hunk 5]
```diff
@@ -123,44 +123,82 @@ This function requires a token ID as its argument and returns the associated URI
...
 
-- **The easiest and cheapest way to upload to Arweave**. Akord makes it simple to upload to Arweave via the [API](https://docs.akord.com/api-and-dev-tools/quickest-way-to-upload-to-arweave), [CLI](https://docs.akord.com/api-and-dev-tools/build/cli) or [app](https://v2.akord.com/signup). The only service to offer free Arweave storage, 100 MB on [signup](https://v2.akord.com/signup), their prices are also the [cheapest in the ecosystem](https://akord.com/pricing).
+Upload your file to the vault and wait for it to be successfully committed on the Arweave blockchain, normally 5-15 minutes. 
 
-- **A secure space to manage on-chain data**. Akord’s advanced system of end-to-end encryption not only secures your files and messages, but uses key rotation to facilitate collaboration within private vaults.
-  
-- **A focus on user experience for developers and end users**. Akord’s vision is to make the power of web3 accessible to everyone, a vision shared by Kadena.
 
-### Use Akord
+**1b. Upload with the Akord API**
```
Comment (lsgunnlsgunn) [inline packages/apps/docs/src/docs/build/nft-marmalade/storage.md:143]:
```suggestion
### Upload with the Akord API
```

File: packages/apps/docs/src/docs/build/nft-marmalade/storage.md [hunk 6]
```diff
@@ -123,44 +123,82 @@ This function requires a token ID as its argument and returns the associated URI
...
 
-All users get [100 MB of free permanent storage on Arweave](https://v2.akord.com/signup) when signing up, it literally takes a minute. 
+- [First, create your account in seconds](https://v2.akord.com/signup) (100 MB free to test your workflow)
```
Comment (lsgunnlsgunn) [inline packages/apps/docs/src/docs/build/nft-marmalade/storage.md:145]:
```suggestion
1. Create your account in seconds](https://v2.akord.com/signup) (100 MB free to test your workflow).
```

File: packages/apps/docs/src/docs/build/nft-marmalade/storage.md [hunk 7]
```diff
@@ -123,44 +123,82 @@ This function requires a token ID as its argument and returns the associated URI
...
+
+- [Next, get your API key here](https://v2.akord.com/account/developers)
```
Comment (lsgunnlsgunn) [inline packages/apps/docs/src/docs/build/nft-marmalade/storage.md:147]:
```suggestion
2. Get your [API key here](https://v2.akord.com/account/developers).
```

File: packages/apps/docs/src/docs/build/nft-marmalade/storage.md [hunk 8]
```diff
@@ -123,44 +123,82 @@ This function requires a token ID as its argument and returns the associated URI
...
+},
+body: data
+});
+```
+That's it! You just uploaded the file to Arweave.
+
+For example responses, API uploads with tags, multipart uploads for larger files, and full documentation on the API, please [check out the Akord API docs here](https://docs.akord.com/).
+
+
+**1c. Upload by with Akord CLI**
```
Comment (lsgunnlsgunn) [inline packages/apps/docs/src/docs/build/nft-marmalade/storage.md:168]:
```suggestion
### Upload with Akord CLI
```

File: packages/apps/docs/src/docs/build/nft-marmalade/storage.md [hunk 9]
```diff
@@ -123,44 +123,82 @@ This function requires a token ID as its argument and returns the associated URI
...
+
+You can also use the Akord CLI to upload by syncing with S3 or a local directory. 
+
+For a complete guide, [check out the Akord CLI docs here](https://docs.akord.com/nft-projects/upload-with-app-api-or-cli/sync-s3-or-local-directory-using-cli).
+
+### 2. Get the Arweave gateway URL
```
Comment (lsgunnlsgunn) [inline packages/apps/docs/src/docs/build/nft-marmalade/storage.md:174]:
```suggestion
### Get the Arweave gateway URL
```

File: packages/apps/docs/src/docs/build/nft-marmalade/storage.md [hunk 10]
```diff
@@ -123,44 +123,82 @@ This function requires a token ID as its argument and returns the associated URI
...
+
+After your assets are uploaded, you’ll need the Arweave gateway URLs. This will be used for the image field in the metadata JSON.
+
+
+**2a. Single NFT**
```
Comment (lsgunnlsgunn) [inline packages/apps/docs/src/docs/build/nft-marmalade/storage.md:179]:
```suggestion
#### Single NFT
```

File: packages/apps/docs/src/docs/build/nft-marmalade/storage.md [hunk 11]
```diff
@@ -123,44 +123,82 @@ This function requires a token ID as its argument and returns the associated URI
...
+![Kadena-Docs-2](https://github.com/kadena-community/kadena.js/assets/57722629/42c4c974-8627-4336-a6dc-a3ea0bff1fb9)
+
+At the top you have two gateway URLs: Akord’s own gateway, akrd.net, and another popular gateway, arweave.net:
+
+![Kadena-Docs-3](https://github.com/kadena-community/kadena.js/assets/57722629/474c44b2-0cb9-4cd8-8f7f-8ac46b4d2ee8)
+
+All gateways follow the same scheme: https://{gateway host}/{tx-id}. On Arweave, the transaction ID is the content identifier. 
+
+
+**2b. NFT Collection**
```
Comment (lsgunnlsgunn) [inline packages/apps/docs/src/docs/build/nft-marmalade/storage.md:192]:
```suggestion
#### NFT collection
```

**Review by lsgunnlsgunn — CHANGES_REQUESTED** [review]
Sorry, I thought this was old one. Can we do something different with the numbering other than 1, 1a, 1b, etc.? It isn't a convention used anywhere else in the doc so it seems a little odd.

**Comment by pascaljb** [comment]
@lsgunnlsgunn totally, happy to go with whatever you think makes sense on the headings in the context of the styles you have. Thanks!

**Comment by lsgunnlsgunn** [comment]
@pascaljb cool. I'll approve and merge these tomorrow. Thanks for your patience!  There's a lot to keep track of with our fledgling docs site 😆.

**Review by lsgunnlsgunn — APPROVED** [review]

**Pull request closed without merging**